Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -17,7 +17,11 @@ public class MixedRealityProjectConfiguratorWindow : EditorWindow
             { MRConfig.ForceTextSerialization, true },
             { MRConfig.VisibleMetaFiles, true },
             { MRConfig.VirtualRealitySupported, true },
+#if UNITY_ANDROID
+            { MRConfig.SinglePass, true },
```

Comment:
Recommend reusing the SinglePassInstancing enum value,

Reply:
Maybe define SinglePass as the same value and mark SinglePassInstancing as obsolete.

Reply:
Not exactly sure what you mean by that. Is it possible to mark an enum value as obsolete?

Reply:
I think @davidkline-ms  is saying make a new enum like
MRConfig.OptimalRenderingPath 

and then these get/set would do the appropriate android macro checking
and deprecrate the MRConfig.SingelPassInstancing. which yes you can put obsolete tag under just that item in the enum

Reply:
The suggestion is to set the value of SinglePass to that of SinglePassInstanced. For example (please do not use these values, they are examples)

```
{
   ...
   SinglePass = 17,
   [Obsolete("SinglePassInstancing is obsolete, use SinglePass instead")]
   SinglePassInstancing = 17,
   ...
}
```